lib/assesments/assessment_2/fi_ass2_rohan/fi_ass2_rohan_login.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+
+class Fia2rohanLogin extends StatelessWidget {
+  const Fia2rohanLogin({super.key});
+  @override
+  Widget build(BuildContext context) {
+    Widget smallSpace = const SizedBox(
+      height: 30.0,
+    );
+    Widget BtnContainer = Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: Container(
+        alignment: Alignment.center,
+        width: 270,
+        height: 50,
+        child: Text(
+          "Login",
+          style: TextStyle(fontWeight: FontWeight.bold),
+        ),
+        decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(25.0),
+            border: Border.all(
+              color: Color(0xff9E9898),
+            )),
+      ),
+    );
+    Widget BtnContainer1 = Padding(
```

Comment:
make a seperate widget / class, and pass them login/sign up, backgroundColor etc as property. Follow DRY (Don't repeat yourself)